extensions/blocks/contact-info/index.js

```diff
@@ -48,6 +48,33 @@ export const settings = {
 	attributes,
 	edit,
 	save,
+	example: {
+		attributes: {},
+		innerBlocks: [
+			{
+				name: 'jetpack/email',
+				attributes: {
+					email: 'hello@jetpack.com',
```

Comment:
I'm not sure we should be using a real email address here, and one that mentions Jetpack? Since the block preview will also appear on WordPress.com, maybe we should avoid mentioning jetpack.com here, and go with something like `email@example.com`? It isn't as fun though.

```suggestion
					email: 'email@example.com',
```

Reply:
Yeah. I think we need to see what we used in the past and go with that.